messaging/src/test/java/org/axonframework/messaging/StubProcessingContext.java

```diff
@@ -17,6 +17,7 @@
 package org.axonframework.messaging;
```

Comment:
Perhaps this file belongs in the unitofwork package...wdyt?

Reply:
Sounds good to me. It was in the wrong location indeed. Nice catch

Reply:
It's been moved, resolving.